salt/states/file.py

```diff
@@ -1200,6 +1214,57 @@ def _shortcut_check(name,
                        'should be. Did you mean to use force?'.format(name)), pchanges
 
 
+def _makedirs(name,
+              user=None,
+              group=None,
+              dir_mode=None,
+              win_owner=None,
```

Comment:
I don't understand here what distinguishes `win_owner` from `user`. Why do these need to be distinct?

Reply:
Maybe if you have a state that is managing a file on both Windows and Linux... Honestly I don't know why. I'm happy to use user. If `win_owner` isn't passed, it will use `user`. But we've been using `win_owner` for a while. Just keeping with the paradigm I guess.

Reply:
I believe it was a design decision Dave and I came up with... but I don't remember the reasons.

Reply:
@UtahDave Do you know why?

Reply:
I just went back and looked through the commit history a little bit. If `win_owner` isn't specified, then `user` is used.

My memory is a little hazy on the specifics since this was committed two years ago, but I believe it's because it was getting too difficult to manage Windows file system permissions by shoehorning it into Salt's  Linux file system permission management functions.